### ISSUE TITLE
scanner sometimes keeps an old election.json

### ISSUE THREAD

**Issue by benadida** [votingworks/vxmail#245]
it seems when switching election definition, it's occasionally keeping the old one around.

**Comment by eventualbuddha** [comment]
I think this would fix it:

```diff
diff --git a/src/importer.ts b/src/importer.ts
index fde9572..d3dab94 100644
--- a/src/importer.ts
+++ b/src/importer.ts
@@ -508,5 +508,6 @@ export default class SystemImporter implements Importer {
   public async unconfigure(): Promise<void> {
     await this.doZero()
     await this.store.reset() // destroy all data
+    this.interpreter.reset()
   }
 }
diff --git a/src/interpreter.ts b/src/interpreter.ts
index afd1daf..e242e02 100644
--- a/src/interpreter.ts
+++ b/src/interpreter.ts
@@ -114,6 +114,7 @@ export interface Interpreter {
     interpretFileParams: InterpretFileParams
   ): Promise<InterpretFileResult>
   setTestMode(testMode: boolean): void
+  reset(): void
 }
 
 interface BallotImageData {
@@ -382,6 +383,11 @@ export default class SummaryBallotInterpreter implements Interpreter {
     this.hmpbInterpreter = undefined
   }
 
+  public reset(): void {
+    this.testMode = undefined
+    this.hmpbInterpreter = undefined
+  }
+
   private async interpretBMDFile(
     election: Election,
     { qrcode }: BallotImageData
```

I don't have time to finish it with tests right now, though.

**Comment by eventualbuddha** [comment]
Fixed by votingworks/module-scan#158, I believe.
